clients/app/src/views/screens/UIKit/UIKit.tsx

```diff
@@ -209,15 +209,15 @@ const UIKit: React.FC = () => {
             />
           </View>
           <View style={styles.rowItem}>
-            <SupportersClubCard
+            <FanClubCard
```

Comment:
This PR is for connect back-end and here is using dummy data, maybe use the back-end data?

Reply:
This usage us in UiKit, thats why I have not change it.

Reply:
got it